unlock-app/src/components/interface/Header.js

```diff
@@ -1,23 +1,29 @@
 import PropTypes from 'prop-types'
-import React, { Component } from 'react'
+import React from 'react'
 import styled from 'styled-components'
 import Icons from './icons'
 
-export default class Header extends Component {
-  render() {
-    return (
-      <TopHeader>
-        <h1>{this.props.title}</h1>
-        <Button><Icons.About /></Button>
-        <Button><Icons.Jobs /></Button>
-        <Button><Icons.Github /></Button>
-      </TopHeader>
-    )
-  }
+export default function Header({ forContent, title }) {
+  return (
+    <TopHeader>
+      {!forContent &&
+        <h1>{title}</h1>
+      }
+      {!!forContent &&
+        <Logo>
+          <Icons.UnlockWordMark />
+        </Logo>
+      }
+      <Button><Icons.About /></Button>
+      <Button><Icons.Jobs /></Button>
+      <Button><Icons.Github /></Button>
+    </TopHeader>
+  )
 }
 
 Header.propTypes = {
   title: PropTypes.string,
+  forContent: PropTypes.bool,
```

Comment:
I'm wondering if this property should have a name that's more about what it does, vs what it's for - we might want to use this form of the header for pages other than content in the future? Your call, but wanted to call it out.

Reply:
I actually tried to think about a good name that would explain what it does but since it may do multiple things I went for `forContent` because this will likely be only for content pages... Thanks!